### PR TITLE
TestID's for e2e automation

### PR DESCRIPTION
**Pull request by nischayv**
#1104

### CODE REVIEW

File: ios/AirGoogleMaps/AIRGoogleMapMarkerManager.m [hunk 1]
```diff
@@ -23,6 +23,8 @@ - (UIView *)view
 //  tapGestureRecognizer.cancelsTouchesInView = NO;
 //  [marker addGestureRecognizer:tapGestureRecognizer];
   marker.bridge = self.bridge;
+  marker.isAccessibilityElement = "YES";
+  marker.accessibilityElementsHidden = "NO";
```
Comment (lelandrichardson) [inline ios/AirGoogleMaps/AIRGoogleMapMarkerManager.m]: 1. Are these supposed to be strings?
2. It seems like we shouldn't set these here, but rather set them when/if people set the `testID` prop. (or other accessibility props if we add them, which would be great if we did)

**Comment by lelandrichardson** [comment]
@nischayv thanks for putting up this PR. Would you mind rebasing? Seems like there are several conflicts.

**Comment by unboundfire** [comment]
@lelandrichardson   

@nischayv and I will be updating this PR soon that will add support for testIDs and accessibilityLabels.  We have gotten accessibilityLabels and testIDs to work on iOS(mapkit)/Android. We haven't figured out how to get testIDs to work with iOS google maps since there overlay objects do not seem to support the `accessibilityIdentifier` property and it seems like google maps is generating there own ids when we set `map.accessibilityElementsHidden = NO`.

**Comment by kirakik** [comment]
@unboundfire any news on this?

**Comment by christopherdro** [comment]
Superseded by https://github.com/react-native-community/react-native-maps/pull/2253/

**Pull request closed without merging**